go-controller/hybrid-overlay/pkg/controller/master_test.go

```diff
@@ -156,6 +157,7 @@ var _ = Describe("Hybrid SDN Master Operations", func() {
 
 		err := app.Run([]string{
 			app.Name,
+			"-loglevel=5",
```

Comment:
instead of doing this for every test case...would it be better to have a wrapper around config.InitConfig where we just force the logging to 5 for tests?

Reply:
maybe? I took a quick look and I didn't see an obvious way.
i'd rather get this merged and fix that for the other tests later than trying to do it here.

Reply:
clearly new to all these things, but I'm really suprised log levels cannot be overridden more easily than this. like a cmdline arg should be possible. what am I missing?

Reply:
the tests we have that are written in `ginkgo` create a new command line application `app := cli.NewApp`.
this inherits all of the various cli flags of the ovn-k8s binary under test. The test and it's assertions are executed from within this command-line application.

so, there is a command line flag and thats `-loglevel=5`.

one test = one cli app = one place we need to add the flag.

Tim's suggestion is that rather than adding this to every test invocation we should have a new function like, `config.InitTestConfig` (which is called in every test) that sets the log level to 5.

While that seems simple in theory, in practice I couldn't find an easy way to do this without making some rather disruptive changes in the `config` package... which is much more work than I'm willing to take on in this PR

I hope that helps @jluhrsen

Reply:
oh it's hard to do because we set the log level in the config itself

Reply:
hmmm, i might have a crazy idea, but will open a new PR for that :)